### DISCUSSION TITLE
Knowing Strimzi Operator Version

### DISCUSSION THREAD

**Discussion by vperi1730** [strimzi/strimzi-kafka-operator#4263] [General]
Hi Team,

Just a quick check, After deploying Kafka if I want to check the version of the Kafka app and Operator version. Can we know in some way, do we have any shell script inside the /opt/Kafka/bin which can give these details

Please suggest.

**Comment by scholzj** [comment]
When the operator starts, it prints its version:

```
2021-01-18 01:43:36 INFO  Main:60 - ClusterOperator 0.22.0-SNAPSHOT is starting
```

So you can see it from the logs. Typically, you can also see it from the tag of the container image it is using and by listing the JARs in the image (`ls /opt/strimzi/lib/` for operators, `ls /opt/kafka/libs/` for Kafka) where you find the right JARs and see the version in the filename.

**Comment by vperi1730** [comment]
Sure, thank you for the information.
